syntax = "proto3";
package info;




import "model/inspection/info.proto";

/*
GetInspectionInfo请求
*/
message GetInspectionInfoRequest {
    /*
    套件id
    */
    string pluginId = 1 ;
}

/*
GetInspectionInfoApi返回
*/
message GetInspectionInfoResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    inspection.InspectionInfo data = 4 ;
}

